### PR TITLE
Starting the Lapp chapter

### PR DESCRIPTION
**Pull request by jnmclarty**
Just trying to chip into this effort.  Let me know if you have any feedback.  All contributions are original.

### CODE REVIEW

File: lapp.asciidoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Lightning Applications (LApps)
+
+The lightning network can transmit payments and data.
+This combination enables the establishment of new standards and protocols where the transmission of value and business logic can be strongly coupled.
+Lightning node operators can deploy turing complete code along with their lightning network client of choice.
+
+lApps are developped by combining a set of arbitrary software components with at least one component which interacts with the lightning network protocol.
```
Comment (bitcoina) [inline lapp.asciidoc]: 2 typos.  LApps should have capital L and developed.

**Comment by jnmclarty** [comment]
@bitcoina fixed. lmk if you have any other suggestions.

**Comment by jnmclarty** [comment]
@bitcoina , @aantonop any feedback?  Anything you want me to try to help with?

**Review by GustavoRSSilva — APPROVED** [review]

**Comment by aantonop** [comment]
We will re-open this next year

**Pull request closed without merging**